src/components/Dialogs/Message/AddMessageForm.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
 import { Field, reduxForm } from 'redux-form';
+import { useTranslation } from 'react-i18next';
 
-import { I18N } from '../../../utils/constants';
+import { required, maxLengthCreator } from '../../../utils/validators';
+import { Textarea } from '../../../utils/FormControl/formsControls';
+
+const maxLenght10 = maxLengthCreator(10);
 
 const AddMessageForm = (props) => {
+  const { t } = useTranslation();
```

Comment:
Consider renaming the "t" variable into something more readable.

Reply:
this is the default library behavior